### DISCUSSION TITLE
How to handle pub imports

### DISCUSSION THREAD

**Discussion by Noobware1** [ethanblake4/dart_eval#125] [General]
Is they're a way to handle pub packages other than manually writting bridge code for all of them?

**Comment by ethanblake4** [comment]
Well, the next version of dart_eval (v0.7.0) coming out soon is going to "support" pub imports technically. However, because dart_eval does not support all of Dart's features, most Pub packages still will not work except very simple ones (because dart_eval won't be able to compile them fully). So yes, bridging is proabably the best option because you can bridge just the features you need.

**Comment by Noobware1** [comment]
Writting all the http package and html package as bridge is too much

**Comment by ethanblake4** [comment]
The http package is going to be supported in v0.7.0 probably, it's very simple. I'll look into the html package, it can maybe be supported by version 0.7.5 or so

**Comment by Noobware1** [comment]
Thanks man

**Comment by kodjodevf** [comment]
Hi @ethanblake4, I would like to know if it is possible to compile the http package with the current v0.7.0? if yes, how can I proceed? 
Thank you for your wonderful work

**Reply by ethanblake4** [reply]
No it is not. Still working on it.

**Comment by ethanblake4** [comment]
Update, I'm probably going to release v0.7.0 before it is able to compile `package:http` because it is already so much better than v0.6.5 and fixes so many issues. Unfortunately the http package is a little bit more complicated than I thought, but I'm still doing my best to get it working soon.

**Comment by Zverik** [comment]
The `http` package is bridged in https://pub.dev/packages/flutter_map_eval
